app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryDetailOverviewFragment.kt

```diff
@@ -346,4 +376,438 @@ class ContentEntryDetailOverviewFragment: UstadDetailFragment<ContentEntryWithMo
+    onClickDelete: () -> Unit = {},
+    onClickManageDownload: () -> Unit = {},
+    onClickTranslation: () -> Unit = {},
+    onClickContentJobItem: () -> Unit = {},
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+    )  {
+
+        item {
+            ContentDetails(
+                uiState = uiState
+            )
+        }
+
+        if (uiState.contentEntryButtons?.showDownloadButton == true){
+            item {
+                Button(
+                    onClick = onClickDownload,
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    colors = ButtonDefaults.buttonColors(
+                        backgroundColor = colorResource(id = R.color.primaryColor)
+                    )
+                ) {
+                    Text(stringResource(R.string.download).uppercase(),
+                        color = contentColorFor(
+                            colorResource(id = R.color.primaryColor))
+                    )
+                }
+            }
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        if (uiState.contentEntryButtons?.showOpenButton == true){
+            item {
+                Button(
+                    onClick = onClickOpen,
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    colors = ButtonDefaults.buttonColors(
+                        backgroundColor = colorResource(id = R.color.primaryColor)
+                    )
+                ) {
+                    Text(stringResource(R.string.open).uppercase(),
+                        color = contentColorFor(
+                            colorResource(id = R.color.primaryColor))
+                    )
+                }
+            }
+        }
+
+        items(
+            uiState.activeContentJobItems.size
+        ){
+            ContentJobListItem(
+                uiState = uiState,
+                onClickContentJobItem = onClickContentJobItem,
+                index = it
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        if (uiState.locallyAvailable) {
+            item {
+                LocallyAvailableRow()
+            }
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            QuickActionBarsRow(
+                uiState = uiState,
+                onClickMarkComplete = onClickMarkComplete,
+                onClickDelete = onClickDelete,
+                onClickManageDownload = onClickManageDownload
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
```

Comment:
This should be removed. Not sure where this from the design. Also we should not be using hardcoded strings instead of stringResource.